Gemfile

```diff
@@ -1,48 +1,35 @@
 # frozen_string_literal: true
 
 source 'https://rubygems.org'
+git_source(:github) { |repo| "https://github.com/#{repo}" }
```

Comment:
Doesn't look like this is referenced anywhere. I have a feeling it's added by default?

Reply:
It is used on the `supplejack_common` gem. Line 22 :)